### ISSUE TITLE
Suggestion: pause data stream on page blur

### ISSUE THREAD

**Issue by brentax** [emojitracker/emojitrack-web#13]
If a user selects a different browser tab or minimizes the browser, there's no point in continuing to send that client data until the page is focused again.

**Comment by ericlaw1979** [comment]
Yeah, I think the Page Visibility API is the simple/cheap fix here: https://github.com/mroth/emojitracker/issues/43. You could probably pause DOM updates immediately, then after some period (60s?) have the client disconnect the stream until/unless the page becomes active again.
